.idea/modules.xml

```diff
@@ -2,9 +2,9 @@
 <project version="4">
   <component name="ProjectModuleManager">
     <modules>
```

Comment:
.idea folder should not be commited. Please delete

Reply:
Done